xml/System.ServiceModel.Syndication/SyndicationTextInput.xml

```diff
@@ -80,9 +115,16 @@
         <ReturnType>System.ServiceModel.Syndication.SyndicationLink</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the url of the <c>textInput</c> feed item.</summary>
+        <value>The url of the <c>textInput<c> feed item as a <see cref="T:System.ServiceModel.Syndication.SyndicationLink" />.</value>
```

Comment:
This will fix the broken build

```suggestion
        <value>The url of the <c>textInput</c> feed item as a <see cref="T:System.ServiceModel.Syndication.SyndicationLink" />.</value>
```